product_manufacturer/tests/test_product_manufacturer.py

```diff
@@ -0,0 +1,129 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests.common import SavepointCase
+
+
+class TestProductManufacturer(SavepointCase):
```

Comment:
@FrancoMaxime Could you please change to TransactionCase ?

Reply:
done